specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2016-05-01/Updates.json

```diff
@@ -0,0 +1,383 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Update operation endpoints and objects.",
+    "title": "UpdateAdminClient",
+    "version": "2016-05-01"
+  },
+  "host": "adminmanagement.local.azurestack.external",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.Update.Admin/operations": {
+      "get": {
+          "x-ms-examples": {
+              "Returns a list of support REST operations.": {
+                  "$ref": "./examples/Operations/List.json"
+              }
+          },
+          "description": "Returns a list of support REST operations.",
+          "tags": [
+              "Update"
+          ],
+          "operationId": "Operations_List",
+          "parameters": [
+              {
+                  "$ref": "#/parameters/ApiVersionParameter"
```

Comment:
This ref needs to be fixed, see failure [here](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/364293753).

Reply:
Done.